src/components/CustomerQuota/CheckoutUnsuccessful/CheckoutUnsuccessfulCard.tsx

```diff
@@ -0,0 +1,179 @@
+import React, {
+  FunctionComponent,
+  useContext,
+  useEffect,
+  useState,
+} from "react";
+import { View, StyleSheet, ActivityIndicator } from "react-native";
+import { CustomerCard } from "../CustomerCard";
+import { AppText } from "../../Layout/AppText";
+import { color, size } from "../../../common/styles";
+import { sharedStyles } from "../sharedStyles";
+import { FontAwesome } from "@expo/vector-icons";
+import { useTranslate } from "../../../hooks/useTranslate/useTranslate";
+import { Cart } from "../../../hooks/useCart/useCart";
+import { DarkButton } from "../../Layout/Buttons/DarkButton";
+import { AuthContext } from "../../../context/auth";
+import { ShowFullListToggle } from "../ShowFullListToggle";
+import { usePastTransaction } from "../../../hooks/usePastTransaction/usePastTransaction";
+import { Quota } from "../../../types";
+import { TransactionsGroup } from "../TransactionsGroup";
+import { AlertModalContext } from "../../../context/alert";
+import {
+  groupTransactionsByCategory,
+  sortTransactions,
+  getLatestTransactionTime,
+} from "../NoQuota/NoQuotaCard";
```

Comment:
Hmm, how can we improve on this? You're importing from soma other module and this particular component should not need to know about the NoQuotaCard